pom.xml

```diff
@@ -17,6 +17,23 @@
 		<java.version>11</java.version>
 	</properties>
 	<dependencies>
+		<!--<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jpa</artifactId>
+		</dependency>-->
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-data-jdbc</artifactId>
```

Comment:
There is bid difference between "spring-boot-starter-jdbc" and "spring-boot-starter-**data**-jdbc". We use only jdbc but not Data. Please delete this annotation and you will see that nothing is working.

Reply:
ive changed in new commit